Source: Debian https://sources.debian.org/src/libwmf/0.2.8.4-12/debian/patches/05_gdk-pixbuf-loader-dir.patch/
Upstream: Unknown
Reason: Install gdk-pixbuf-loader into correct location

--- configure.ac
+++ configure.ac
@@ -745,7 +745,7 @@ GDK_PIXBUF_DIR=""
 if test $LIBWMF_BUILDSTYLE != lite; then
 	PKG_CHECK_MODULES(GDK_PIXBUF,gdk-pixbuf-2.0 >= 2.1.2,[
 		GTK_VERSION=`$PKG_CONFIG --variable=gtk_binary_version gtk+-2.0`
-		GDK_PIXBUF_DIR="gtk-2.0/$GTK_VERSION/loaders"
+		GDK_PIXBUF_DIR="gdk-pixbuf-2.0/$GTK_VERSION/loaders"
 		wmf_gdk_pixbuf=yes
 	],[	wmf_gdk_pixbuf=no
 	])
